Lecture3.md

```diff
@@ -137,6 +149,9 @@ run :-  write('Enter the first number: '), read(FirstNumber),
 
 ?- length([1,2,3, ahmad], X).
 
-?- append([1,2,[a, b], X).
+?- append([1,2,[a, b]], [X, X2, X3]). %append/2 (merge 2 lists)
+%   [1,2,[a,b],X,X2,X3]
 
+?- append([[1,2,4], [5, 6, 7]]). %append/1 (list of lists)
```

Comment:
@Rami-Sabbagh, this append/1 won't work in command line, and I think there is no append/1 out there. I suggest you to replace append/1 and append/2 with this append/3 example `append([1,2],[a, b], X).`

Reply:
Oh right my mistake, `append/1` is not for lists.